components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice.go

```diff
@@ -522,9 +522,19 @@ func constructBatchCheckResponse(checkTriggerRespMap map[string][]models.CheckTr
 	// Constructing response which is needed by the handler
 	result := constructResult(ipMap)
 
+	isPassed := true
+	for _, v := range result {
+		testForNode := v.Tests
```

Comment:
do you need this line? You can range it over v.Tests 
for _, checksResult := range v.Tests {

Reply:
Done